packages/core/src/lib/Log.ts

```diff
@@ -14,10 +14,45 @@
  *  limitations under the License.
  */
 
+/**
+ * An interface for loggers.
+ */
+export interface Log {
+
+  verbose: boolean;
+
+  /**
+   * Prints a debug message to the Log. message is ignored if the Log is not set to verbose.
+   * @param message the message the be printed.
+   * @param args extra arguments for the console.
+   */
+  debug(message: string, ...args: string[]): void;
+
+  /**
+   * Prints an info message to the Log. message is ignored if the Log is not set to verbose.
```

Comment:
These comments seem to be wrong (the bit about verbose only applies to `debug` I think).

Also, capital letters at the start of sentences please.

Reply:
I didn't write it, and I think it is right, if `verbose == false` it will not print anything